var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -86,84 +108,58 @@ class Qt(Package):
           working_dir='qtscript',
           when='@5.0:5.12 %gcc@8.3:')
 
+    # Build-only dependencies
     depends_on("pkgconfig", type='build')
-    # Use system openssl for security.
-    depends_on("openssl@:1.0", when='@:5.9')
-    depends_on("openssl")
-    depends_on("glib", when='@4:')
-    depends_on("gtkplus", when='+gtk')
+    depends_on("flex", when='+webkit', type='build')
+    depends_on("bison", when='+webkit', type='build')
+
+    # Dependencies, then variant- and version-specific dependencies
+    depends_on("icu4c")
+    depends_on("jpeg")
+    depends_on("libmng")
+    depends_on("libtiff")
     depends_on("libxml2")
     depends_on("zlib")
-    depends_on("dbus", when='@4:+dbus')
-    depends_on("libtiff")
+    depends_on("freetype", when='freetype=system')
+    depends_on("gperf", when='+webkit')
+    depends_on("gtkplus", when='+gtk')
+    depends_on("openssl", when='+ssl')
+    depends_on("sqlite", when='+sql')
+
     depends_on("libpng@1.2.57", when='@3')
-    depends_on("libpng", when='@4:')
-    depends_on("libmng")
-    depends_on("jpeg")
-    depends_on("icu4c")
-    depends_on("fontconfig", when=(sys.platform != 'darwin'))  # (Unix only)
-    depends_on("freetype")
-    depends_on("sqlite")
     depends_on("pcre+multibyte", when='@5.0:5.8')
-    depends_on("pcre2+multibyte", when='@5.9:')
-    depends_on("double-conversion", when='@5.7:')
-    depends_on("harfbuzz", when='@5:')
-
-    # Core options:
-    # -doubleconversion  [system/qt/no]
-    # -iconv             [posix/sun/gnu/no] (Unix only)
-    # -pcre              [system/qt]
-
-    # Gui, printing, widget options:
-    # -harfbuzz          [system/qt/no]
-    # -xkbcommon-x11     [system/qt/no]
-    # -system-xkbcommon
-
-    # Database options:
-    # -sqlite            [system/qt]
-
-    # Qt3D options:
-    # -assimp            [system/qt/no]
-
-    # QtQml
-    depends_on("python", when='@5.7.0:', type='build')
```

Comment:
This PR updates the `Python` package but at first glance appears to remove the python dependency. Is it pulled in transitively somewhere?

Reply:
This was an unintentional deletion. Good catch!